go.mod

```diff
@@ -1,20 +1,22 @@
 module github.com/openshift/library-go
 
-go 1.22.0
+go 1.22.4
 
-toolchain go1.22.1
+toolchain go1.23.0
 
 require (
 	github.com/RangelReale/osincli v0.0.0-20160924135400-fababb0555f2
 	github.com/blang/semver/v4 v4.0.0
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
-	github.com/distribution/distribution/v3 v3.0.0-20230511163743-f7717b7855ca
+	github.com/distribution/distribution/v3 v3.0.0-beta.1
```

Comment:
This seems to be a pre-release: https://github.com/distribution/distribution/releases/tag/v3.0.0-beta.1

Reply:
Yes, but even being a beta, it is better than `v3.0.0-20230511163743-f7717b7855ca` which is not even alpha. It was depending of a random commit sha from the master branch instead of a released (git tag) one.